src/expression.d

```diff
@@ -5312,6 +5312,9 @@ public:
                         return e.semantic(sc);
                     }
                 }
+                // ti is an instance which requires IFTI.
+                sds = ti;
+                type = Type.tvoid;
```

Comment:
This in particular changed the semantic of `typeof(func!arg)` from `Error: expression (func!arg) has no type` to plain `void`. I think it's incorrect to type a not fully instantiated template function as void (even though there is some precedence w/ templates being of type void).
For non-function templates it's an error when arguments are missing and even with this change `typeof(&func!arg)` triggers the same error.
On top of this all this change broken `is(typeof(T.someAttribute))` detection for types w/ opDispatch.